test/new-e2e/pkg/runner/configmap.go

```diff
@@ -26,6 +26,8 @@ const (
 	AgentAPPKey = commonconfig.DDAgentConfigNamespace + ":" + commonconfig.DDAgentAPPKeyParamName
 	// AgentPipelineID pulumi config parameter name
 	AgentPipelineID = commonconfig.DDAgentConfigNamespace + ":" + commonconfig.DDAgentPipelineID
+	// AgentMajorVersion pulumi config parameter name
+	AgentMajorVersion = commonconfig.DDAgentConfigNamespace + ":" + commonconfig.DDAgentMajorVersion
```

Comment:
**thought**:
Should we already extend for minor and point versions?

Reply:
That's actually not really needed, at least for now. This parameter is currently used to switch from one "bucket" to the other (we have one for a6 and one for a7). Until now, the a7 bucket was hardcoded si we were not able to pull the a6 artefacts. This is now possible